api/services/JwtService.js

```diff
@@ -32,10 +56,9 @@ module.exports = {
 
   generateIdToken(client, user, scope, nonce) {
     const now = Math.floor(Date.now() / 1000);
-    let sub = user._id;
-    if (client.id === 'iasc-prod' || client.id === 'iasc-dev') {
-      sub = user.email;
-    }
```

Comment:
Removing these lines was the specific fix described in the ticket. The rest is logging.